### PR TITLE
Adding Colin Principe colin.principe@gmail.com

### PR DESCRIPTION
**Pull request by FatherOfCurses**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -332,6 +332,34 @@ module.exports = [
       'Gatsby',
     ],
   },
+  {
+    name: 'Colin Principe',
+    description:
+      'Software Guild graduate, Full stack developer, Java backend, Angular/React frontend.',
+    url: 'http://principe.io/uses',
```
Comment (HugoDF) [inline src/data.js]: This isn't a uses page

Comment (FatherOfCurses) [reply src/data.js]: URL changed, was having issues with my hosting.  Latest commit resolves properly.
[https://github.com/FatherOfCurses/awesome-uses/commit/9d2d15026f52119d3335fbfcca759cad6945ca75](Latest commit)

File: src/uses.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (HugoDF) [inline src/uses.html]: Could we remove this file?

Comment (FatherOfCurses) [reply src/uses.html]: Deleted from my commit.

**Comment by FatherOfCurses** [comment]
Going to cancel this pull request and submit a new one because checks are failing.

**Pull request closed without merging**